README.md

```diff
@@ -31,6 +31,10 @@ In the `env.config.js` file, add the following
 
 The DatadogLoggingService is a concrete implementation of the logging service interface that sends messages to Datadog that can be seen in Datadog RUM and Datadog browser logs. When in development mode, all messages will instead be sent to the console.
 
+When you configure a Datadog logging implementation, you'll set a number of variables used by the DatadogLoggingService. Most (eg. `DATADOG_APPLICATION_ID`, `DATADOG_SESSION_SAMPLE_RATE`) are either supplied to you by Datadog when you set up a RUM application, or are options about how much data we should be saving. Optional variables include:
```

Comment:
nit: While the Datadog RUM application's SDK configuration provides values for `DATADOG_SESSION_SAMPLE_RATE` and other settings, it's not really something that needs to be supplied _from Datadog_ (i.e., you can freely change the value as needed for your application), unlike the `DATADOG_APPLICATION_ID` and `DATADOG_CLIENT_TOKEN` config settings which are specific Datadog-provided values.